src/main/java/Duke.java

```diff
@@ -1,10 +1,100 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final String DOTTED_LINE = "____________________________________\n";
```

Comment:
Good job on allocating a final for the horizontal line string since it appears quite frequently, makes the code very easy to read